src/current/_includes/v24.1/ui/statements-table.md

```diff
@@ -11,7 +11,7 @@ Execution Count | Cumulative number of executions of statements with this finger
 Database | The database in which the statement was executed.
 Application Name | The name specified by the [`application_name`]({{ link_prefix }}show-vars.html#supported-variables) session setting.
 Statement Time | Average [planning and execution time]({{ link_prefix }}architecture/sql-layer.html#sql-parser-planner-executor) of statements with this statement fingerprint within the time interval. <br><br>The gray bar indicates the mean latency. The blue bar indicates one standard deviation from the mean. Hover over the bar to display exact values.
-% of All Runtime | How much time this statement fingerprint took to execute compared to all other statements that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.
+<a id="percent-of-all-runtime"></a>% of All Runtime | How much time this statement fingerprint took to execute in proportion to all other statements (even those not displayed) that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.<br><br>NOTE: The sum of the values displayed for this column may not equal 100%. The percentage for each fingerprint is calculated by dividing the numerator consisting of the fingerprint's runtime by the denominator consisting of the sum of the runtimes for all statement fingerprints in the time interval. By “all statements fingerprints”, this means all user statement fingerprints (not only the displayed top <i>k</i> fingerprints based on the [search criteria](#search-criteria), but also statements executed within the time period not included in the displayed result set), as well as internal statement fingerprints that are never included in the displayed result set. The search criteria is applied subsequent to the calculation of `% of All Runtime`.
```

Comment:
Same suggestion as earlier

Reply:
Incorporated some suggestions, kept some original sentences to maintain accuracy of explanation.